.github/fabricbot.json

```diff
@@ -712,29 +712,13 @@
         "requireAllStatuses": false,
         "conditionalMergeTypes": [
           {
-            "mergeType": "squash",
+            "mergeType": "merge",
             "condition": {
               "placeholder": "labels",
-              "operator": "notContains",
+              "operator": "contains",
```

Comment:
So this changes it to use a merge when the do-not-squash-merge label is on the PR?